guides/AboutAPIKeys.md

```diff
@@ -11,41 +11,45 @@ API keys are authentication tools that account owners and co-owners can create a
 
 ## Create and use API keys
 
-1. Create or request an API key:
-
-    - If you have direct access to the site, create a key with the relevant permissions in the [API Keys Manager](https://manage.wix.com/account/api-keys). Collect the key and the account ID.
-    - If you don't have direct access to the site, request the key and the account ID from the site owner.  
-
-2. Call the Site API's **Query Sites** endpoint to collect all the account's associated site IDs.
-
-      > **Note:**
-      > The site ID for a current site can be obtained from the site URL in your browser.
-      > For example, the site ID appears after the '/dashboard/' part of this URL: <br />
-      > <br />
-      > ![site Id in URL](../media/siteid.png)
-
-      1. Enter the API key in the authorization header. Note that the API key does not need to be refreshed.  
-      2. Enter the account ID in the account ID header, as shown below. (In other calls, the site ID header may be required. Refer to the documentation for each specific call.)
-
-    A complete header for an API request looks like this:
-
-     ```sh
-     curl <GET/POST> \
-       '<endpoint>' \
-       -H 'Authorization: <APIKEY>' \
-       -H 'wix-account-id: <ACCOUNTID>' \
-       -H 'wix-site-id: <SITEID>'
-     ```
-
-3. Make all following calls using the account ID and/or site ID, as required.
+### Step 1 | Create or request an API key:
+- If you have direct access to the site, create a key with the relevant permissions in the [API Keys Manager](https://manage.wix.com/account/api-keys). Collect the key and the account ID.
+- If you don't have direct access to the site, request the key and the account ID from the site owner.  
+
+### Step 2 | Call the Site API's **Query Sites** endpoint to collect all the account's associated site IDs.
+
+  > **Note:**
+  > The site ID for a current site can be obtained from the site URL in your browser.
+  > For example, the site ID appears after the '/dashboard/' part of this URL: <br />
+  > <br />
+  > ![site Id in URL](../media/siteid.png)
+
+  1. Enter the API key in the authorization header. Note that the API key does not need to be refreshed.  
+  2. Calls may require either the account ID header or the site ID header. 
```

Comment:
this is good info, but it feels weird to put it as a step when they are calling a specific endpoint and we know which one they'll need in this case

Reply:
oh, this helps a lot. Only reason I have "Steps" was because of this subsection numbered list. But you're right, it's not really steps. So I can flatten it back to the way it was and adjust here. Thanks.